### PR TITLE
prepping database and routing

### PR DESCRIPTION
**Pull request by songtv-0006**
prepping database

### CODE REVIEW

**Comment by songtv-0006** [comment]
ready

**Comment by songtv-0006** [comment]
ready

**Comment by songtv-0006** [comment]
ready

File: app/Models/Task.php [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = ['name'];
```
Comment (tranducquoc) [inline app/Models/Task.php]: $fillable = [
    'name',
];

File: app/Models/Task.php [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    /**
+     * Get the user that own the task.
+     */
+    public function user()
+    {
+        return $this->belongsTo('App\Models\User');
```
Comment (tranducquoc) [inline app/Models/Task.php]: return $this->belongsTo(User::class);

File: app/Models/User.php [hunk 3]
```diff
@@ -36,4 +36,12 @@ class User extends Authenticatable
     protected $casts = [
         'email_verified_at' => 'datetime',
     ];
+
+    /**
+     * Get all of the tasks for the user.
+     */
+    public function tasks()
+    {
+        return $this->hasMany('App\Models\Task');
```
Comment (tranducquoc) [inline app/Models/User.php]: fix tương tự

File: routes/web.php [hunk 4]
```diff
@@ -13,4 +15,4 @@
 
 Route::get('/', function () {
     return view('welcome');
-});
+});
```
Comment (tranducquoc) [inline routes/web.php]: end line

**Comment by songtv-0006** [comment]
ready

File: app/Models/Task.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+
+namespace App\Models;
+
+use App\Models\User;
+use Illuminate\Database\Eloquent\Model;
+
+class Task extends Model
+{
+    protected $fillable = [
+    'name',
```
Comment (tranducquoc) [inline app/Models/Task.php]: 4 space em ơi

**Comment by songtv-0006** [comment]
ready

**Comment by tranducquoc** [comment]
merged

**Pull request merged**